src/main/webapp/user.html

```diff
@@ -21,15 +21,10 @@
     </nav>
     <a href="createUsername.html">Create a Username Here!</a>
 
-
-      <div id="email"></div>
-      <div id="form"></div>
-
-      <div id="login-container"></div>
       
       <div id="Setup">Here is a list of reviews you have made! </div>
       <div class= "inline" id="email"></div>
-      <ul id="list"></ul>
+      <div id="list"></div>
```

Comment:
Does this div need to contain something?

Reply:
no